core/transformer/CHANGELOG.md

```diff
@@ -0,0 +1,291 @@
+# Change Log - @itwin/core-transformer
```

Comment:
pkgs that moved out of lockstep shouldnt be re-creating these files since they make no sense